test/e2e/v3/generate_test.go

```diff
@@ -139,6 +139,7 @@ func GenerateV3(kbc *utils.TestContext, crdAndWebhookVersion string) {
 		"--project-version", "3",
 		"--domain", kbc.Domain,
 		"--fetch-deps=false",
+		"--component-config=true",
```

Comment:
I think we need a new test only for the componentConfig so that we can ensure this scenario

Reply:
yes, will remove this too.

Reply:
Removed.